### PR TITLE
ZCS-6097: Fixed locators for fixing HSM test cases

### PR DESCRIPTION
**Pull request by pallavikhairnar21-zz**
Fixed locators for fixing HSM test cases

### CODE REVIEW

File: src/java/com/zimbra/qa/selenium/projects/admin/pages/PageZextrasHSM.java [hunk 1]
```diff
@@ -27,23 +27,23 @@
...
 		public static final String SECONDARY_VOLUME_DELETE_BUTTON = "css=div[id='ztabv__ZxPowerstore_zawiz_top_grouper_2'] td[class='ZWidgetTitle']:contains('Delete')";
-		public static final String HSM_POLICY_ADD_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_14'] td[class='ZWidgetTitle']:contains('Add')";
-		public static final String HSM_POLICY_EDIT_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_14'] td[class='ZWidgetTitle']:contains('Edit')";
-		public static final String HSM_POLICY_DELETE_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_14'] td[class='ZWidgetTitle']:contains('Delete')";
-		public static final String lastPolicyInList = "css=div.hsmList div[class='DwtListView-Rows'] div:last-child";
+		public static final String HSM_POLICY_ADD_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_16'] td[class='ZWidgetTitle']:contains('Add')";
+		public static final String HSM_POLICY_EDIT_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_16'] td[class='ZWidgetTitle']:contains('Edit')";
+		public static final String HSM_POLICY_DELETE_BUTTON = "css=div[id='ztabv__ZxPowerstore_group_16'] td[class='ZWidgetTitle']:contains('Delete')";
```
Comment (miteshsavani) [inline src/java/com/zimbra/qa/selenium/projects/admin/pages/PageZextrasHSM.java:40]: Need to update the locators.
div id value contains number at the end ('ztabv__ZxPowerstore_group_16') which will changed on every login

File: src/java/com/zimbra/qa/selenium/projects/admin/pages/WizardAddNewVolume.java [hunk 2]
```diff
@@ -26,13 +26,13 @@
 public class WizardAddNewVolume extends AbsWizard {
 	
 	public static class Locators {
-		public static final String VOLUME_TYPE_LOCAL = "css=[style*='z-index: 7'] input[id='zdlgv__DLG_NEW_PWRSTR_VOLUME_BObject_49']";
+		public static final String VOLUME_TYPE_LOCAL = "css=input[id='zdlgv__DLG_NEW_PWRSTR_VOLUME_BObject_63']";
```
Comment (miteshsavani) [inline src/java/com/zimbra/qa/selenium/projects/admin/pages/WizardAddNewVolume.java]: Update this locator, it contains number in the ID at the end

File: src/java/com/zimbra/qa/selenium/projects/admin/pages/PageZextrasHSM.java (hunk 1 above)
Comment (pallavikhairnar21-zz) [reply src/java/com/zimbra/qa/selenium/projects/admin/pages/PageZextrasHSM.java:40]: I have verifiied 'ztabv__ZxPowerstore_group_16' locator. It is constant and not changing changing on every login.

File: src/java/com/zimbra/qa/selenium/projects/admin/pages/WizardAddNewVolume.java (hunk 2 above)
Comment (pallavikhairnar21-zz) [reply src/java/com/zimbra/qa/selenium/projects/admin/pages/WizardAddNewVolume.java]: I have verified locator. It is constant and not changing changing on every login.

**Review by miteshsavani — APPROVED** [review]

**Pull request merged**